libs/sdk-ui-geo/src/core/geoChart/helpers/geoChart/data.ts

```diff
@@ -55,15 +63,45 @@ export function getLocation(latlng: string | null): IGeoLngLat | null {
     };
 }
 
+export function parseCoordinate(coordinate: string | null): number | null {
```

Comment:
Almost the same method as `getLocation` above, just without parsing of the latitude and longitude from `latitude;longitude` string. Possibly could be consolidated.